blocks/components/editable/index.js

```diff
@@ -75,6 +78,32 @@ export default class Editable extends wp.element.Component {
 		this.props.onChange( this.getContent() );
 	}
 
+	isStartOfEditor() {
+		const range = this.editor.selection.getRng();
+		if ( range.startOffset !== 0 || ! range.collapsed ) {
+			return false;
+		}
+		const start = range.startContainer;
+		const body = this.editor.getBody();
+		let element = start;
+		do {
+			const child = element;
+			element = element.parentNode;
+			if ( element.childNodes[ 0 ] !== child ) {
```

Comment:
>`element.firstChild`? (It's faster)

I agree with the suggestion but now I'm intrigued in what way it would be faster? Seems like the difference is array access at an index, which I wouldn't expect to be slow.